docs/actions/http.md

```diff
@@ -1,58 +1,25 @@
-# HTTP
+# GET
```

Comment:
We could add an example of a complete valid http target defined in json and in kotlin

Reply:
we may address that in another PR for details on env & target configuration.
I just wanted this to be merge quickly for CSS style and light template so we could go on documenting actions